test/test_brownian.py

```diff
@@ -146,33 +149,35 @@ def _eval(key):
     assert pval > 0.1
 
 
-@pytest.mark.parametrize("levy_area", ["", "space-time"])
-@pytest.mark.parametrize("use_levy", (False, True))
-def test_conditional_statistics(levy_area, use_levy):
+def conditional_statistics(
+    levy_area: _LA, use_levy: bool, tol=2**-6, spacing=2**-6, spline: _Spline = "sqrt"
+):
     key = jr.PRNGKey(5678)
     bm_key, sample_key, permute_key = jr.split(key, 3)
-
     # Get >80 randomly selected points; not too close to avoid discretisation error.
     t0 = 0.3
     t1 = 8.7
     ts = jr.uniform(sample_key, shape=(100,), minval=t0, maxval=t1)
+    # ts = jnp.array([1.0, 3.0, 6.0, 7.0])
     sorted_ts = jnp.sort(ts)
+    ts = []
     prev_ti = sorted_ts[0]
-    ts = [prev_ti]
+    ts.append(prev_ti)
     for ti in sorted_ts[1:]:
-        if ti < prev_ti + 2**-10:
+        if ti < prev_ti + spacing:
             continue
         prev_ti = ti
         ts.append(ti)
     ts = jnp.stack(ts)
-    assert len(ts) > 80
+    assert len(ts) > min(0.2 * (8.0 / spacing), 75)  # for spacing = 2**-5, this is 51
```

Comment:
maybe just provide a lower bound as an explicit argument to this function, rather than using a heuristic like this?

Reply:
Good idea.